Add show_banner trait to control the banner display (#6808)
jupyter/notebook: docs/source/frontend_config.rst

``<section>`` can take various values like ``notebook``, ``tree``, and
``editor``. A ``common`` section contains configuration settings shared by all
pages.

Persisting configuration settings
---------------------------------

> A banner is shown to the user to inform him about news or updates. This 

Reviewer: Minor suggestion:

```suggestion
A banner might be shown to users to inform them about news or updates. This 
```
